### PR TITLE
#844: docs: update GET docs

### PR DESCRIPTION
**Pull request by sebzz2k2**

### CODE REVIEW

**Comment by sebzz2k2** [comment]
#844

**Review by lucifercr07 — APPROVED** [review]

**Comment by lucifercr07** [comment]
@sebzz2k2 please rebase the PR once and update with master.

File: docs/src/content/docs/commands/GET.md [hunk 1]
```diff
@@ -4,39 +4,48 @@ description: The `GET` command in DiceDB is used to retrieve the value of a spec
 ---
 
 The `GET` command in DiceDB is used to retrieve the value of a specified key. If the key exists, the value is written and
-if it does not then the command returns `nil`. This is one of the most fundamental operations in DiceDB.
+if it does not then the command returns `nil` and an error is returned if the value stored at key is not a string, because GET only handles string values. This is one of the most fundamental operations in DiceDB.
```
Comment (JyotinderSingh) [inline docs/src/content/docs/commands/GET.md:7]:
```suggestion
if it does not then the command returns `nil` and an error is returned if the value stored at key is not a string. This is one of the most fundamental operations in DiceDB.
```

**Review by JyotinderSingh — APPROVED** [review]

**Comment by JyotinderSingh** [comment]
LGTM. Thanks for contributing @sebzz2k2!

**Pull request merged**